src/content/docs/guides/fields/_partials/_slider.mdx

```diff
@@ -25,8 +25,8 @@ const thumbData = useThumbMetadata(0);
       <Thumb />
     </div>
 
-    <div v-if="errorMessage" v-bind="errorMessageProps" class="error">
-      {{ errorMessage }}
+    <div v-bind="errorMessageProps" class="error">
+      <div v-if="errorMessage" class="error-message">{{ errorMessage }}</div>
```

Comment:
I don't think you need the inner div here, do you?

Reply:
Yep we don't need it since the `errorMessage` is an empty string when there aren't any errors, I will fix that